pkg/state/state_nv_ipam_cni.go

```diff
@@ -62,8 +63,8 @@ type stateNVIPAMCNI struct {
 
 type nvipamRuntimeSpec struct {
 	runtimeSpec
-	// is true if cluster type is Openshift
-	IsOpenshift bool
+	IsOpenshift     bool
+	CniBinDirectory string
```

Comment:
here its in runntimeSpec while the reset is in the *RenderData ?
is there a particular reason ? should be align with the rest of the  states  ?

Reply:
Addressed. Now all CNIs use a common `cniRuntimeSpec`